### PR TITLE
[13.0][IMP] stock_analytic: Prevent error when set real_time valuation and stock.valuation.layer is auto-created 

### PR DESCRIPTION
**Pull request by victoralmau**
Prevent error when set **real_time** valuation and `stock.valuation.layer` is auto-created  (according to (according to products in test category).

Please @pedrobaeza and @joao-p-marques can you review it?

@Tecnativa TT29728

### CODE REVIEW

**Review by joao-p-marques — APPROVED** [review]

**Review by pedrobaeza — COMMENTED** [review]
Can you please remove totally demo data for avoiding more side effects?

**Comment by victoralmau** [comment]
Changes done.

File: stock_analytic/tests/test_stock_picking.py [hunk 1]
```diff
@@ -59,7 +56,7 @@ def setUp(self):
         self.outgoing_picking_type = self.env.ref("stock.picking_type_out")
         self.incoming_picking_type = self.env.ref("stock.picking_type_in")
 
-        self.product_categ.update(
+        self.product_categ = self.env.ref("product.product_category_5").copy(
```
Comment (pedrobaeza) [inline stock_analytic/tests/test_stock_picking.py]: This is still using demo data as "template", so the side effects are the same: any modification on such demo data will affect here. Please create the product and the category from scratch.

Comment (victoralmau) [reply stock_analytic/tests/test_stock_picking.py]: Ok sorry, changes done.

**Comment by joao-p-marques** [comment]
@pedrobaeza seems like the bot got stuck... Maybe reissue the command?

**Pull request merged**